students/njschafi/Lesson08/assignment/inventory.py

```diff
@@ -0,0 +1,39 @@
+"""Inventory Module for Lesson08 Assignment"""
+import csv
+from functools import partial
+#pylint: disable=E0602, W0613, C0103
+
+def add_furniture(invoice_file='', customer_name='', item_code='',

```

Comment:
Avoid using default values for parameters, use them only when there are usage cases in which default values make sense to the code. In this case, calling _add_furniture()_ with no parameters will make the function crash:

```
>>> import inventory as inv
>>> inv.add_furniture()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ldconejo/Documents/UW/SP_Python220B_2019/SP_Python220B_2019/students/njschafi/Lesson08/assignment/inventory.py", line 10, in add_furniture
    with open(invoice_file, "a+", newline='') as csvfile:
FileNotFoundError: [Errno 2] No such file or directory: ''
```

Reply:
I updated the code so add_furniture() can handle no parameters. Updated code is in https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/pull/60